uniparc-rest/src/main/java/org/uniprot/api/uniparc/request/UniParcIdsDownloadRequest.java

```diff
@@ -0,0 +1,35 @@
+package org.uniprot.api.uniparc.request;
+
+import lombok.Data;
+
+import org.uniprot.api.rest.request.IdsSearchRequest;
+import org.uniprot.api.rest.validation.ValidDownloadByIdsRequest;
+import org.uniprot.store.config.UniProtDataType;
+
+/**
+ * @author sahmad
+ * @created 28/07/2021
+ */
+@Data
+@ValidDownloadByIdsRequest(accessions = "upis", uniProtDataType = UniProtDataType.UNIPARC)
+public class UniParcIdsDownloadRequest implements IdsSearchRequest {
+    private String upis;
+    private String fields;
+    private String download;
+    private Integer size;
+    private String cursor;
+
+    public String getCommaSeparatedIds() {
+        return this.upis;
+    }
+
+    @Override
+    public String getFacetFilter() {
+        return null;
+    }
+
+    @Override
+    public String getFacets() {
```

Comment:
I think we need to accept facets, when download= false, right?
If you agree, we can not return always null

Reply:
Same argument as facetFilter.